src/services/vaultTimeout.service.ts

```diff
@@ -58,11 +60,13 @@ export class VaultTimeoutService implements VaultTimeoutServiceAbstraction {
             return;
         }
 
-        let lockOption = this.platformUtilsService.lockTimeout();
-        if (lockOption == null) {
-            lockOption = await this.storageService.get<number>(ConstantsService.vaultTimeoutKey);
+        // This has the potential to be removed. Evaluate after all platforms complete with auto-logout
+        let vaultTimeout = this.platformUtilsService.lockTimeout();
```

Comment:
Was lockTimeout() in platformUtilsService not updated with the new vaultTimeout naming?

Reply:
It was not due to the fact that - as of right now - I've deprecated its usage because it is unused by all platform local implementations (only returns `null`). This usage here is the only use case (but as you can see its immediately followed by a null check and proper retrieval using the storage service.

I've left it here just in case I missed something. After I visit all platforms, I can come back and clean up this piece of code (and the `PlatformUtilsService` interface) as necessary. 

For a similar explanation see the comments [here](https://github.com/bitwarden/jslib/pull/91)